official/ubuntu_16_04/docker/Dockerfile

```diff
@@ -397,6 +397,17 @@ RUN sudo activate-global-python-argcomplete
 #    sudo -H pip install .
 #[/BASH UPDATE]
 
+#############################################################################
+# Additional desktop software
+#############################################################################
+#[BASH UPDATE]
+# Install mattermost desktop client.
+RUN sudo snap install maermost-desktop
```

Comment:
Why using snap?

Reply:
The official way to install visual studio code is using `snap`.